src/java/org/apache/cassandra/config/ParameterizedClass.java

```diff
@@ -31,7 +31,7 @@
 import static org.apache.cassandra.utils.Shared.Scope.SIMULATION;
 
 @Shared(scope = SIMULATION)
-public class ParameterizedClass implements Serializable
```

Comment:
Also need to remove import at line 22

Reply:
Removed
Interestingly, `ant checkstyle;ant checkstyle-test` passed somehow 🤔